src/søknad/steg/3-barnadine/BarnaDine.less

```diff
@@ -1,118 +1,118 @@
 .barna-dine {
-    width: 620px;
-    margin: 0 auto;
+  width: 620px;
+  margin: 0 auto;
 
-    .informasjonstekst {
-        font-weight: bold;
-        background: none;
-        border: none;
-        margin-bottom: 1rem;
+  .informasjonstekst {
+    font-weight: bold;
+    background: none;
+    border: none;
+    margin-bottom: 1rem;
 
-        .alertstripe__tekst {
-            font-weight: bold;
-        }
+    .alertstripe__tekst {
+      font-weight: bold;
     }
+  }
+
+  .hjelpetekst {
```

Comment:
Er denne scopet innenfor barna-dine komponenten? Hvis ikke kan det potensielt være en kilde til bug kanskje? Med mindre dere selv har laget hjelpetekst komponenten :)

Reply:
Er lagt til hjelpetekst styling for både spørsmål og barnadine, yikes 😱  
Men jeg mekka en Hjelpetekst komponent nå, og flyttet styling koden inn dit 👩‍🔧